src/odemis/util/dataio.py

```diff
@@ -189,3 +201,20 @@ def _split_planes(data):
         das.append(plane)
 
     return das
+
+def open_acquisition(filename, fmt=None):
+    if fmt:
```

Comment:
Add a description of this function. (including arguments and return type: list of DataArray or DataArrayShadow)